packages/website/docusaurus.config.js

```diff
@@ -40,7 +40,7 @@ const config = {
   tagline: "API first with Cadl for Azure services",
   url: "https://microsoft.github.io",
   baseUrl: process.env.CADL_WEBSITE_BASE_PATH ?? "/",
-  onBrokenLinks: "throw",
+  onBrokenLinks: "log",
```

Comment:
Do we want to keep this?

Reply:
I suspect bug in docusaurus. The reported link in md was correctly set. Opened an investigation bug to track this. #1574

Reply:
Sounds good, thanks.